docs/agile-methodologies.md

```diff
@@ -3,11 +3,25 @@ id: agile-methodologies
 title: Agile Methodologies
 sidebar_label: Agile Methodologies
 ---
+import useBaseUrl from '@docusaurus/useBaseUrl';
 
 ## What are agile methodologies?
 
 These are work methodologies that allow adaptation to the circumstances of the projects and reduce the documentation processes, they are generally used in projects that do not have a defined scope with certainty, which causes them to be very prone to change, because in these methodologies deliveries of functional products in a short time (2 to 4 weeks) allow the process of changes and correction of errors to be fast and not have a very high cost for the project in general.
 
+## Modern Agile
```

Comment:
Lets use an H3 tag and change the title to  Modern Agility 

`### Modern Agility`